.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+---
+# https://docs.corelightning.org/docs/release-checklist
+name: "Release 🚀"
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+[0-9a-z]+'
```

Comment:
what's an example of what this last regex is looking to match? 

it seems like it'd ignore things such as `v0.1.3-beta`. I believe the regex that you'd want for that would need to include the `.` and `-` in the options clause? Untested but...

`v[0-9]+.[0-9]+[0-9a-z.-]+`

Reply:
We literally append rcX, so "24.11rc3"...